roles/nginx/defaults/main.yml

```diff
@@ -0,0 +1,6 @@
+---
+migration_plan_name: nginx-migplan-{{ ansible_date_time.epoch }}
+migration_name: nginx-mig-{{ ansible_date_time.epoch }}
+namespace: nginx-example
```

Comment:
It may be better to use a listed variable here like
```
namespaces:
- nginx-example
```
and then use it in the `mig-plan.yml.j2` as
```
namespaces: "{{ namespaces }}"
```
This will help in the future when we will test a multi-project migration, but it is not so important right now.

Reply:
@Danil-Grigorev Yes, I'm thinking about moving the CR creation to a separate role that can be called to assemble the migration, this would be re-usable by any other tests. I still need to make cases for the mig-controller test scenarios and will merge those changes on that PR.

Reply:
@Danil-Grigorev Implemented changes